### PR TITLE
#696252 - EAPI7 for sys-fs/gt5

### PR DESCRIPTION
**Pull request by Kalich**
Hello, there is EAPI 7 update for package sys-fs/gt5

### CODE REVIEW

**Comment by Kalich** [comment]
Closes: https://bugs.gentoo.org/696252

**Comment by juippis** [comment]
Hey there! 

There are few things we need, that are missing from your contribution. First of all, the git summary we use is like: `sys-fs/gt5: bump to EAPI-7` and then you need to add a sign-off to your commit's message box, with your legal name and e-mail. Check https://gitweb.gentoo.org/repo/gentoo.git/log/ these for example. You can then reference a bug in the commit message, by adding a `Bug: https://bugs.gentoo.org/696252` field in the message box as well. That's a tracker bug, don't close it. 

Check links below for more info about our git workflow, 
https://www.gentoo.org/glep/glep-0066.html#commit-messages
https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin

File: sys-fs/gt5/gt5-1.4.0-r2.ebuild [hunk 1]
```diff
@@ -18,12 +18,17 @@ RDEPEND="|| ( www-client/links
 		www-client/elinks
 		www-client/lynx )"
 
+PATCHES=(
+	"${FILESDIR}"/${P}-bash-shabang.patch
+	"${FILESDIR}"/${P}-empty-dirs.patch
+)
+
+src_prepare() {
```
Comment (juippis) [inline sys-fs/gt5/gt5-1.4.0-r2.ebuild]: Looks like you can just remove this phase from this ebuild.

**Comment by thesamesam** [comment]
gt5 was handled later on after this PR.

**Pull request closed without merging**